doc/nrf/ug_nrf5340.rst

```diff
@@ -496,6 +496,40 @@ Follow these steps to build and program a multi-image build to the nRF5340 appli
 
 #. Program the application sample using :guilabel:`Target` > :guilabel:`Download zephyr/merged.hex`.
 
+Using |VSC| extension
+======================
+
+You can build and program separate images or combined images using |VSC|.
+
+Separate images
+---------------
+
+To build and program only the application core, follow the instructions in :`Building an application`_ and use ``nrf5340dk_nrf5340_cpuapp`` or ``nrf5340dk_nrf5340_cpuapp_ns`` as build target.
+
+To build and program a dedicated network sample, follow the instructions in `Building an application`_ and use ``nrf5340dk_nrf5340_cpunet`` as the build target.
+
+.. _ug_nrf5340_VSC_multi_image:
+
+Multi-image build
+-----------------
+
+If you are working with Bluetooth LE, Thread, Zigbee, or Matter samples, the network core sample is built as child image when you build the application core image (see :ref:`ug_nrf5340_multi_image` above).
```

Comment:
```suggestion
If you are working with Bluetooth LE, Thread, Zigbee, or Matter samples, the network core sample is built as a child image when you build the application core image (see :ref:`ug_nrf5340_multi_image` above).
```

Reply:
I think this is still missing the article `a`?

Reply:
Its there :)

Reply:
I am also not seeing the "a" there. I have just refreshed my browser several times but it is still missing. Maybe GitHub is not showing the changes correctly, but can you double check just in case? I noticed there was a similar change on line 372, so can you confirm you are also seeing it on line 518 as well?

![Screenshot 2022-05-18 143158](https://user-images.githubusercontent.com/17974570/169039343-e60706c0-9166-4096-8180-7154c3c50099.png)
.